sbnanaobj/StandardRecord/SRCRTHitMatch.h

```diff
@@ -17,7 +17,15 @@ namespace caf
       SRCRTHitMatch();
       virtual ~SRCRTHitMatch() {}
       SRCRTHit hit; ///< The CRT hit
-      float distance; ///< Distance from the projected TPC track to the CRT hit [cm]
+      float distance; ///< Distance from the extrapolated TPC track to the CRT hit [cm]
+      int region; ///< region of the matched crt plane
+      int sys; ///< system of the matched crt hit (e.g. ICARUS: Top 0, Side 1, Bottom 2)
```

Comment:
Maybe better to make this into an enum? For example, in SBND we track North/South/East/West walls - see SREnums.h:`enum SBNDCRTTagger_t`.

Reply:
Hi @kjplows I can do that, but keep in mind that a similar object (int) is also used by SBND here: https://github.com/SBNSoftware/sbnobj/blob/8f98cbb03bde70452077737022f8cff71f4360c4/sbnobj/Common/CRT/CRTHit.hh#L36C2-L37C1
the CRTHit object is common to both ICARUS and SBND.
Let me know if you still want this moved to an enum and I can do that

Reply:
I think for now it's fine to keep it as an int, but at some point it'll likely be better to move this to an enum. Thanks for the clarification!